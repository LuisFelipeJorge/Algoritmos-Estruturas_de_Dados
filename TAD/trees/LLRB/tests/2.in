16
1 6
1 3
1 12
1 1
1 2
1 4
1 15
2 15
3 1
2 6
3 6
4
5
8
7
6