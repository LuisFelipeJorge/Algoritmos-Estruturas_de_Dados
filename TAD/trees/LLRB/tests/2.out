erro
erro
12
4
15 
1 
1 3 4 2 12 15 6 
1 2 3 4 6 12 15 
6 2 1 4 3 15 12 
